platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -143,57 +140,47 @@ BUILD SUCCESSFUL in 1s
 
-Thank you to [Matthew Von-Maszewski](https://github.com/matthewvon) for the contribution.
+#### Root variant exposed by the Resolution Result API
 
-<a name="core-plugin-improvements"></a>
-### Core plugin improvements
+The [`ResolutionResult` API](javadoc/org/gradle/api/artifacts/result/ResolutionResult.html), which contains the results of dependency resolution, now exposes the root variant of the resolved graph in addition to its owning component.
 
-Gradle provides [core plugins](userguide/plugin_reference.html) for build authors, offering essential tools to simplify project setup and configuration across various languages and platforms.
+Previously, the API only exposed the root component, which exposed the first-level dependencies as well as dependencies from other selected variants in the root component:
 
-#### Configuration cache compatibility for Swift and C++ plugins
+```groovy
+def rootComponent = configurations.runtimeClasspath.incoming.resolutionResult.rootComponent
+```
 
-The following Swift and C++ plugins are now [configuration cache](userguide/performance.html#enable_configuration_cache) compatible: 
-- [Swift application](userguide/swift_application_plugin.html)
-- [Swift library](userguide/swift_library_plugin.html)
-- [XCTest](userguide/xctest_plugin.html)
-- [C++ application](userguide/cpp_application_plugin.html)
-- [C++ library](userguide/cpp_library_plugin.html)
-- [CppUnit](userguide/cpp_unit_test_plugin.html)
-- [GoogleTest](userguide/cpp_testing.html)
-- [Visual Studio](userguide/visual_studio_plugin.html)
-- [XCode](userguide/xcode_plugin.html)
+In this release, the root variant is also exposed.
+The root variant is a synthetic variant representing the `Configuration` being resolved, and exposes the first-level dependencies of a resolution:
 
+```groovy
+def rootVariant = configurations.runtimeClasspath.incoming.resolutionResult.rootVariant
+```
 
-#### Build Init plugin enables configuration cache in generated projects
+The updated API allows for more precise traversal of dependency graphs by operating at the variant level instead of the component level.
+For example, by traversing dependencies at the variant level, it is possible to differentiate between the production code of a component and its test fixtures.
 
-All projects generated by the [Build Init](userguide/build_init_plugin.html) plugin now have configuration cache enabled in the root `gradle.properties` file.
-This also includes projects converted from Maven.
+For more details on how to perform a graph traversal, see [Programmatic Dependency Resolution](userguide/programmatic_dependency_resolution.html).
 
-<!-- ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-ADD RELEASE FEATURES ABOVE
-==========================================================
+#### Provider support for configuration extensions in the Configuration API
```

Comment:
Do we know if this will make it into 8.11?

Reply:
I asked Tom and it seems like a strong maybe so I left it as a TODO. This will be addressed during my review with Louis closer to release.